integration-test/test.sh

```diff
@@ -4,7 +4,7 @@ IFS=$'\n\t'
 
 # these flighting values should match Makefile
 readonly TEST_SUBS_AZURE="c3dfd792-49a4-4b06-80fc-6fc6d06c4742"
-readonly TEST_REGION_AZURE="Brazil South"
+readonly TEST_REGION_AZURE="Central US EUAP"
```

Comment:
Is this for actually publishing the extension for testing? (If so then using EUAP is fine, otherwise, we might want to facilitate external development by using a public region?)